src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -215,61 +124,17 @@ define(function (require, exports, module) {
         },
 
         render: function () {
-            var doc = this.props.document;
-
-            var layerComponents = doc.layers.allVisibleReversed
-                    .filter(function (layer) {
-                        // Do not render descendants of collapsed layers unless
```

Comment:
I think this optimization got dropped. Was that a conscious choice or an accident?